src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -33,14 +22,964 @@ to make sure IE does not go into compatibility mode.
+
+```
+try {
+  aFunctionThatMightFail();
+} catch (err) {
+  Sentry.captureException(err);
+}
+```
+
+{% capture __alert_content -%}
+It's possible to throw strings as errors. In this case, the Sentry SDK will not record tracebacks.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="warning"
+%}
+
+&nbsp;
+### Automatically Capturing Errors
+By including and configuring the Sentry Browser SDK, Sentry will automatically attach global handlers to capture uncaught exceptions and unhandled rejections.  
+
+[{% asset js-index/automatically-capture-errors.png alt="Stack trace of a captured error" %}]({% asset js-index/automatically-capture-errors.png @path %})
+
+&nbsp;
+### Automatically Capturing Errors with Promises
+By default, Sentry for JavaScript captures unhandled promise rejections as described in the official ECMAScript 6 standard.
+
+Configuration may be required if you are using a third-party library to implement promises:
+
+Most promise libraries have a global hook for capturing unhandled errors. You may want to disable default behavior by setting `captureUnhandledRejections` option to `false` and manually hook into such event handler and call `Sentry.captureException` or `Sentry.captureMessage` directly.
+
+For example, the [RSVP.js library](https://github.com/tildeio/rsvp.js/) (used by Ember.js) allows you to bind an event handler to a [global error event](https://github.com/tildeio/rsvp.js#error-handling).
+
+```
+RSVP.on('error', function(reason) {
+  Sentry.captureException(reason);
+});
+```
+
+[Bluebird](http://bluebirdjs.com/docs/getting-started.html) and other promise libraries report unhandled rejections to a global DOM event, `unhandledrejection`. In this case, you don't need to do anything; we've already got you covered with the default `captureUnhandledRejections: true` setting.
+
+Please consult your promise library documentation on how to hook into its global unhandled rejection handler, if it exposes one.
+
+&nbsp;
+## Manually Trigger Errors
+If your app is in a broken state, but error handling isn't throwing explicit errors, you can manually trigger errors.
+
+**[Mimi note: Trying to cater to the kind of dev who has a set flow. Usually, 1. Initialize, 2. optionally configure, 3. Manually trigger an error (because you wanna take it out for a test drive)]**
+
+&nbsp;
+### Capturing Errors
+
+The most common form of capturing is to capture errors. In general, if you have something that looks like an exception it can be captured.
+
+For more information, see [Capturing Errors / Exceptions](#capturing-errors).
+
+&nbsp;
+### Using Custom Error Types
+If you want to define your own error types, you can generate a custom error with `new MyAppError('message')` and by creating a class that extends the built in `Error()`. You can also attach additional data to your error instance and Sentry will parse it along with your error, just like it does with regular errors.
```

Comment:
How about:

```suggestion
If you want to define your own error types, you can do that by creating a class that [extends the built-in `Error` constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#Custom_Error_Types) and instantiating it's new instance, for example `new MyAppError('message', { fatal: true })`. You can also attach additional data to your error instance and Sentry will parse it along with your error, just like it does with regular errors.
```

Reply:
Something along that lines is easier to grasp imo.

Reply:
I like how that links to a standard.